vitest.config.js

```diff
@@ -39,6 +39,10 @@ export default defineConfig({
     exclude: [...configDefaults.exclude, "examples/**"],
   },
 
+  resolve: {
+    conditions: mode === "test" ? ["browser"] : [],
+  },
+
```

Comment:
can we add a comment explaining the reason for this?

Reply:
Done!